### PR TITLE
Signup: Update post processing to include email confirmation instructions

### PR DESCRIPTION
**Pull request by shaunandrews**
The PR aims to update the post-signup processing screen. The new design ditches the step-by-step progress indicator in favor of a pulsing button. You still know something is happening, but there's no need to focus on the details.

Since you have to wait here for a few seconds, I added an interesting (and subtle) background animation.

We also think that users with confirmed email addresses may be more likely to return again, so we display a notice asking you to check your email.

processing screen

### CODE REVIEW

**Comment by coreh** [comment]
Added the user's email address to the message, and a check so that we don't display this to verified users. What are we going to show in place of the confirmation prompt? Looks very empty:

https://cldup.com/SgePuDcygu.png

**Comment by coreh** [comment]
@shaunandrews fixed the email display, added the “click to proceed” functionality, and reverted a bunch of changes I made early on that were not the best way of doing things.

I feel like the wait time is too short for the animation to start playing, and for the user to actually read the message about email confirmation. Should we increase it a little bit? Try it and tell me what you think.

Also please share your opinion on the wording/copy after the wait is complete.

**Comment by shaunandrews** [comment]
@coreh I tried the branch a few times in Safari and Chrome and in both cases, after submitting step 5 (creating my account) I'm not taking to the processing screen — instead I see a disabled step 5 again.

**Comment by shaunandrews** [comment]
@coreh Awesome. I made some quick tweaks the copy. Feels good, thanks for the great work on this one.

> I feel like the wait time is too short for the animation to start playing, and for the user to actually read the message about email confirmation. Should we increase it a little bit?

Yea, sometimes the wait is perfect, but sometimes its super quick. A minimal artificial delay of 4-5 seconds may help.

File: client/signup/processing-screen/index.jsx [hunk 1]
```diff
@@ -131,12 +131,12 @@ module.exports = React.createClass( {
...
 
 					{ this.props.loginHandler
-						?	<p className="signup-process-screen__title">We're all set.</p> :
+						?	<p className="signup-process-screen__title"><strong>Done!</strong> Thanks for waiting, we're ready for you to get started. </p> :
```
Comment (coreh) [inline client/signup/processing-screen/index.jsx]: Needs i18n

**Comment by apeatling** [comment]
I still find the wording here a little awkward. "It'll only take a few seconds" then "Just a minute". I'm also wondering why this is a pulsing button, do you now click this to go to the next step (I haven't tried it).

**Comment by shaunandrews** [comment]
> I still find the wording here a little awkward. "It'll only take a few seconds" then "Just a minute".

The wording has changed, and matches the image in the original description at the top.

>  I'm also wondering why this is a pulsing button, do you now click this to go to the next step (I haven't tried it)?

It pulses while we do out thing in the background. Once ready, it turns into a normal button that you then click to proceed.

**Comment by coreh** [comment]
The translations have landed! (#5676)

Once this is reviewed it should be good to go.

File: client/signup/step-header/style.scss [hunk 2]
```diff
@@ -9,14 +9,14 @@
...
+	//font-weight: 200;
 	margin: 0 0 5px 0;
 }
 
 .step-header__subtitle {
-	color: $gray;
+	color: darken( $gray, 20 );
 	font-size: 14px;
-	font-weight: 300;
+	//font-weight: 300;
```
Comment (mtias) [inline client/signup/step-header/style.scss]: Is this supposed to be like this or removed?

Comment (coreh) [inline client/signup/step-header/style.scss]: Fixed.

**Comment by michaeldcain** [comment]
:+1:

**Comment by apeatling** [comment]
@shaunandrews Looks good. One issue I found going through is that when you hit continue, it is saying that you're ready to start, but if you've selected a domain you go straight to the checkout form. It feels anti-climactic. 

Can we change the wording if your next step is to pay for a plan? Or maybe we can just not use the button if they still have to pay?

**Comment by apeatling** [comment]
Having done this a couple of times now, I feel a bit frustrated by the button. It's another manual step, and I'm not sure of the reasons why. It is a bit of a jolt to the flow.

**Comment by alisterscott** [comment]
@coreh: I've raised the issue I discussed as #6114

**Pull request merged**